src/renderer/pages/TorrentListPage.js

```diff
@@ -220,6 +196,29 @@ module.exports = class TorrentList extends React.Component {
           {downloadIcon}
         </i>
       )
+
+      if (TorrentPlayer.isPlayableTorrentSummary(torrentSummary)) {
+        playButton = (
+          <i
+            key='play-button'
+            title={playTooltip}
+            className={'button-round icon play ' + playClass}
```

Comment:
Shame on me, thought I was so clever with my last second code reordering.
Thanks for fixing!!!